Automattic/jetpack
# ThreatsDataViews: Update Extension filter

## Description
Replaces the `Extension` filter with a more specific set of options - `Plugin` and `Theme`.

## Proposed changes:

* Filter out plugins and themes in the extensions block of the primary threats data computation.

### Other information:


## Jetpack product discussion




## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch
* Load storybook with `cd projects/js-packages/storybook && npm run storybook:dev`
* View the Threats Data View component
* Ensure the list filter includes a Plugin and Theme option (will need to modify the story data in order to render the Theme filter) when a threats exists for either and that Extension is no longer an option
* Verify that applying the filter works
* Ensure no regressions are introduced

## Screenshots
![Screen Shot 2024-11-04 at 12 57 25](https://github.com/user-attachments/assets/5f377a61-0599-4dee-a211-44a9f592709f)
